src/css/Fonts/raleway.css

```diff
@@ -0,0 +1,20 @@
+@font-face {
+  font-family: 'ralewayregular';
+  src: url('Raleway-Regular.ttf') format('truetype');
```

Comment:
Are there a woff version available for all fonts? It just makes browser compatibility easier